wordpress-mobile/gutenberg-mobile
# Move BlockManager to gutenberg

Step 8 of https://github.com/wordpress-mobile/gutenberg-mobile/issues/958
This is the next step in our port of the BlockList to gutenberg.

Gutenberg PR https://github.com/WordPress/gutenberg/pull/16239

### Testing Instructions
- Boot the app on iOS and Android
- Check that the block list works as expected.

Update release notes:
